examples/udp_client/linux_udp_client.cc

```diff
@@ -0,0 +1,135 @@
+/**************************************************************************/ /**
+ * @brief Simple Linux UDP client example.
+ * @file
+ *
+ * @note
+ * This is a minimal UDP client implementation, meant as an example of how to
+ * connect to a device and decode incoming data. It is not robust to network
+ * outages, socket reconnects, or other typical network errors. Production
+ * implementations must check for and handle these cases appropriately (by
+ * checking expected `errno` values, etc.).
+ ******************************************************************************/
+
+#include <cerrno>
+#include <cmath> // For lround()
+#include <csignal> // For signal()
+#include <cstdio> // For fprintf()
+#include <cstring> // For memcpy()
+#include <string> // For stoi() and strerror()
+
+#include <netdb.h> // For gethostbyname() and hostent
+#include <netinet/in.h> // For IPPROTO_* macros and htons()
+#include <sys/socket.h> // For socket support.
+#include <unistd.h> // For close()
+
+#include <iostream>
+#include <fstream>
+#include <arpa/inet.h>
+
+#include <point_one/fusion_engine/parsers/fusion_engine_framer.h>
+
+#include "../common/print_message.h"
+
+#define DEBUG_ON 0
```

Comment:
This doesn't seem to be used anymore - remove?

Reply:
I think this debug statement should stay. Only because as the user of Atlas this helped me debug issues (usually on the server side) about the network topology between the Atlas and the listening linux client. Of course you are the owner of this codebase so it is your decision. But I'd say if you want to make the user's life easier... leave it.

Reply:
Also, just thinking out load.. this also shows the developer that it is possible to receive packets from different ip addresses on the same port.

Reply:
Oh and also, the compiler would also complain about unused variables (e.g. `their_ip`).

Reply:
If you found it helpful, I'm ok with leaving it. Just see the notes below about using `#if` instead, and about not printing out `buffer`.

On the unused variables note, technically `recvfrom()` accepts `nullptr` for the `srcaddr` and `addrlen` arguments, so if you are not using them you don't need to specify them, or you can use `recv()` instead. See the note in https://man7.org/linux/man-pages/man2/recv.2.html.

Reply:
done.